src/lib.rs

```diff
@@ -130,24 +130,6 @@ pub trait IdAllocator: BaseAllocator {
     fn available(&self) -> usize;
 }
 
-#[inline]
-const fn align_down(pos: usize, align: usize) -> usize {
```

Comment:
These functions are designed to be used not only by `mod bitmap` but also by other modules, so they should not be moved into `bitmap.rs`.

Reply:
`pub(crate) fn` won't prohibit dead code lints. So the option is one of those
* `pub fn`
* `pub fn` with `#[doc(hidden)]` to not expose the internal API
* `#[allow(dead_code)]`

I prefer `#[allow(dead_code)]`.